packages/presentations/20-11-18-Rokas-Lengvenis-browser-CPR/deck.mdx

```diff
@@ -0,0 +1,141 @@
+import { Head } from 'mdx-deck'
+
+export { default as theme } from './theme'
+
+<Head>
+  <title>Critical Rendering Path</title>
+</Head>
+
+# Browser CRP
+
+---
+
+## Talk plan
```

Comment:
I think you could just promote these points to be the main subject of this slide.